recipes/repeatmasker/patch1

```diff
@@ -11,48 +11,16 @@
  
  #
  # Get the date
-@@ -549,26 +549,26 @@
- # Verify that the libraries exist
- #
- die "RepeatMasker could not find the repeat library at:\n"
--    . "  $RepeatMaskerConfig::REPEATMASKER_DIR/Libraries/RepeatMaskerLib.embl\n"
-+    . "  $RepeatMaskerConfig::REPEATMASKER_LIB_DIR/RepeatMaskerLib.embl\n"
-     . "This file is necessary to run RepeatMasker.  A minimal version of this\n"
-     . "file is distributed with RepeatMasker.  For an extensive library of\n"
-     . "interspersed repeats please obtain the complete file at:\n"
-     . "http://www.girinst.org and install before using RepeatMasker.\n\n"
-     unless (
--       -s "$RepeatMaskerConfig::REPEATMASKER_DIR/Libraries/RepeatMaskerLib.embl"
--       || -s "$RepeatMaskerConfig::REPEATMASKER_DIR/Libraries/Dfam.hmm" );
-+       -s "$RepeatMaskerConfig::REPEATMASKER_LIB_DIR/RepeatMaskerLib.embl"
-+       || -s "$RepeatMaskerConfig::REPEATMASKER_LIB_DIR/Dfam.hmm" );
+@@ -537,7 +537,7 @@
+ 
+ my ( $combLib, $rmLibraryVersion, $dbversion ) =
+     LibraryUtils::validateLibraries(
+-                              "$RepeatMaskerConfig::REPEATMASKER_DIR/Libraries",
```

Comment:
I know this got merged already, but maybe we should consider using `REPEATMASKER_LIB_DIR` instead of `REPEATMASKER_DIR/Libraries`?